document/src/tests/documentidtest.cpp

```diff
@@ -127,26 +102,18 @@ void DocumentIdTest::testCopying()
     DocumentId id3("doc:ns:foo");
     id3 = id2;
 
-    CPPUNIT_ASSERT_EQUAL(id1, id2);
-    CPPUNIT_ASSERT_EQUAL(id1, id3);
-}
-
-void
-DocumentIdTest::testParseId()
-{
-    // Moved to base/documentid_test.cpp
+    EXPECT_EQ(id1, id2);
+    EXPECT_EQ(id1, id3);
 }
 
-void
-DocumentIdTest::checkNtnuGlobalId()
+TEST(DocumentIdTest, checkNtnuGlobalId)
 {
     DocumentId id("doc:crawler:http://www.ntnu.no/");
-    CPPUNIT_ASSERT_EQUAL(vespalib::string("gid(0xb8863740be14221c0ac77896)"),
+    EXPECT_EQ(vespalib::string("gid(0xb8863740be14221c0ac77896)"),
```

Comment:
Consider updating indentation on the following line (or removing linebreak)

Reply:
Fixed